lisa/analysis/idle.py

```diff
@@ -72,6 +73,50 @@ def signal_cpu_active(self, cpu):
         # Fix sequences of wakeup/sleep events reported with the same index
         return handle_duplicate_index(cpu_active)
 
+    @memoized
```

Comment:
not sure if we want to memoize things that return series or dataframe as it leads to high memory usage, even if the user of the trace will not look at that df anymore.
https://github.com/ARM-software/lisa/pull/1235 will bring a better way of doing that, avoiding excessive memory usage and swapping the df to disk if that's faster than recomputing it.